zboxcore/sdk/sdk.go

```diff
@@ -76,6 +75,80 @@ func GetLogger() *logger.Logger {
 	return &Logger
 }
 
+type storageSdkSchema struct {
+	sdkInitialized bool
+}
+
+func NewStorageSdkSchema() *storageSdkSchema {
+	return &storageSdkSchema{}
+}
+
+var storageSdk = NewStorageSdkSchema()
+
+type StorageSdkSchema interface {
+	GetNetwork() *Network
+	SetMaxTxnQuery(num int) error
+	SetQuerySleepTime(time int) error
+	CreateReadPool() (err error)
+	// GetReadPoolInfo for given client, or, if the given clientID is empty,
+	// for current client of the sdk.
+	GetReadPoolInfo(clientID string) (info *AllocationPoolStats, err error)
+	// ReadPoolLock locks given number of tokes for given duration in read pool.
+	ReadPoolLock(dur time.Duration, allocID, blobberID string, tokens, fee int64) (err error)
+	// ReadPoolUnlock unlocks tokens in expired read pool
+	ReadPoolUnlock(poolID string, fee int64) (err error)
```

Comment:
Need to follow the single way to return errors `(err error)` or `error` for all functions